homeassistant/components/alexa/smart_home.py

```diff
@@ -1416,6 +1415,39 @@ def temperature_from_object(hass, temp_obj, interval=False):
     return convert_temperature(temp, from_unit, to_unit, interval)
 
 
+def _convert_ha_mode(ha_mode):
+    mode = next(
+        (k for k, v in API_THERMOSTAT_MODES.items() if ha_mode in v),
+        None
+    )
+
+    return mode
+
+
+def _thermostat_context_from_entity(hass, entity, mode=None, temp=None):
+    unit = hass.config.units.temperature_unit
+    ha_mode = mode if mode\
```

Comment:
Please convert to a normal if-statement if an inline if-statement is wrapping two lines.

Reply:
sure I will